rx/__init__.py

```diff
@@ -43,11 +47,11 @@ def case(mapper: Callable[[], Any],
     .. marble::
         :alt: case
 
-        --1---------------|
-        a--1--2--3--4--|
-         b--10-20-30---|
+        ---1---|
+        a----1----2----3----4---|
+         b----10---20---30---|
         [case(mapper, { 1: a, 2: b })]
-        ---1--2--3--4--|
+        -----1----2----3----4---|
```

Comment:
I think the resulting observable should be shifted to the right (+ 3 hyphens), i.e. the 'a' observable is subscribed when the source observable emits `1`.

Reply:
```
        ---1---|
        a----1----2----3----4---|
         b----10---20---30---|
        [case(mapper, { 1: a, 2: b })]
        --------1----2----3----4---|
```

Do you mean like this? I don't know, I tried to keep the vertical alignment the same... I think the point is to emphasize that the result sequence is the same as `a`. But I got this wrong before :-)